docs/en/Admin-CLI.md

```diff
@@ -91,3 +96,6 @@ write operations on the under storage.
 ```bash
 $ ./bin/alluxio fsadmin ufs --mode readOnly hdfs://ns
 ```
+
+The `fsadmin ufs` subcommand that takes the UFS URI as argument, the argument should be the root
```

Comment:
The `fsadmin ufs` subcommand takes a UFS URI as an argument. The argument should be a root UFS URI like ...